smk/gene_abundance.smk

```diff
@@ -592,11 +592,19 @@ rule gene_abund_compute:
     shell:
         """
         time (
+            # Random sleep to avoid choking NFS
```

Comment:
Maybe the thread count or RAM could be increased to limit the number of parallel jobs hammering the NFS.

Reply:
I have tried to handle this in the past via many different routes, but the best one in the end is reporting the actual needs and asking the user to restrict the number of tasks via `--jobs`. Otherwise, we are blocking unused memory and cores from other users of the queuing system. It's only hammering the NFS for a minute or two, and then starts working on local disk.

Reply:
I don't know why snakemake doesn't have a per-rule job limit. Instead the user is expected to create a pseudo-resource to limit the jobs...

Reply:
I just googled and found this - not a bad idea:
https://stackoverflow.com/questions/51977436/restrict-number-of-jobs-by-a-rule-in-snakemake